ansible/roles/mordred/files/create_roles_tenant.py

```diff
@@ -220,31 +221,52 @@ def args_parser():
     parser.add_argument("-a", "--anonymous",
                         action="store_true",
                         help="Create bap_anonymous_access role")
+    parser.add_argument("-f", "--force",
+                        action="store_true",
+                        help="Overwrite role and tenant")
     args = parser.parse_args()
 
     return args
 
 
-def create_roles(anonymous, opensearch_url, tenant):
+def create_roles(anonymous, opensearch_url, tenant, force=False):
     for role in ROLES_MAPPING:
         if not anonymous and role == "bap_tenant_anonymous_access_role":
             continue
 
         role_name = role.replace("tenant", tenant)
+        if not force and exist(opensearch_url, "role", role_name):
+            print("{}: already exists".format(role_name))
+            continue
         data = ROLES_MAPPING[role]
         data = data.substitute(tenant=tenant)
         base_url = opensearch_url + API_ROLES + role_name
         response = run_requests("PUT", base_url, data=data)
         print("{}: {}".format(response['status'], response['message']))
 
 
-def create_tenant(opensearch_url, tenant):
+def create_tenant(opensearch_url, tenant, force=False):
+    if not force and exist(opensearch_url, "tenant", tenant):
+        print("{}: already exists".format(tenant))
+        return
     base_url = opensearch_url + API_TENANT + tenant
     data = '{"description": "Tenant created automatically."}'
     response = run_requests("PUT", base_url, data=data)
     print("{}: {}".format(response['status'], response['message']))
 
 
+def exist(url, type, name):
```

Comment:
I think it will be better if for `type` you declare an `Enum` and then you pass the value that way and not with a string. Also you can compare both values and raise an error if non is valid.